src/mapblock_mesh.cpp

```diff
@@ -1389,9 +1329,8 @@ void MapBlockMesh::updateCameraOffset(v3s16 camera_offset)
 {
 	if (camera_offset != m_camera_offset) {
 		translateMesh(m_mesh, intToFloat(m_camera_offset-camera_offset, BS));
-		if (m_enable_vbo) {
+		if (m_enable_vbo)
```

Comment:
I don't really think these braces needed to be removed because they were used in a consistent manner. But don't put them back, I'm just making a comment :)